PowerShell/JumpCloud Commands Gallery/Windows Commands/Windows - Set ReleaseChannel In JumpCloud Password Manager.md

```diff
@@ -0,0 +1,35 @@
+#### Name
+
+Windows - Set JumpCloud Password Manager's Release Channel | v1.0 JCCG
+
+#### commandType
+
+windows
+
+#### Command
+
+```
+# Set $RELEASE_CHANNEL to beta OR dogfood OR public depending on your desired release channel
+$RELEASE_CHANNEL = "public"
+
+#------- Do not modify below this line ------
+
+$FILE_PATH = "$env:APPDATA\JumpCloud Password Manager\data\daemon\releaseChannel.txt"
```

Comment:
I'd encourage you to go run a Windows PowerShell command from JumpCloud and print out the `$ENV:APPDATA` directory. 

The windows command doesn't work as described in the PR.
![Screenshot 2024-07-09 at 8 20 02 AM](https://github.com/TheJumpCloud/support/assets/54448601/d6e1e23f-051a-4e85-b2d3-d667da7b7e4a)

Look at this command and I think you'll see why.
![Screenshot 2024-07-09 at 8 21 23 AM](https://github.com/TheJumpCloud/support/assets/54448601/46796d8c-0e3a-4bd2-ab88-629f14b814b5)